packages/styles/components/_c.select.scss

```diff
@@ -0,0 +1,96 @@
+$select-border-radius : get-border-radius('m');
+$select-border-width : get-border('s');
+
+.mc-select {
+  border: $select-border-width solid $color-select-border;
+  border-radius: $select-border-radius;
+  display: block;
+
+  @include set-font-scale('05');
+
+  background: $color-select-background;
+  color: $color-select-text;
+  position: relative;
+  width: 100%;
+
+  /* autoprefixer: ignore next */
+  -webkit-transform: translateZ(0);
+
+  &:hover,
+  &.is-hover {
+    border: $select-border-width solid $color-select-hover-border;
+  }
+
+  &:focus,
+  &.is-focus {
+    border: $select-border-width solid $color-select-focus-border;
+  }
+
+  &:disabled,
+  &.is-disabled {
+    border: $select-border-width solid $color-select-disabled-border;
+    background: $color-select-disabled-background;
+    cursor: default;
+  }
+
+  &:invalid,
+  &.is-invalid {
+    border: $select-border-width solid $color-select-invalid-border;
+  }
+
+  &:valid,
+  &.is-valid {
+    box-shadow: none;
+    border: $select-border-width solid $color-select-valid-border;
+  }
+
+  &.is-invalid {
+    &:hover:not(:focus) {
+      border: $select-border-width solid $color-select-invalid-hover-border;
+    }
+  }
+
+  &.is-valid {
+    &:hover:not(:focus) {
+      border: $select-border-width solid $color-select-valid-hover-border;
+    }
+  }
+
+  &__input {
+    @include reset-input;
+    @include set-font-scale('05');
+
+    text-overflow: '';
+    text-indent: 0.01px;
+    width: 100%;
+    padding: calc(#{$mu100} - #{$select-border-width});
+    background: transparent;
+    z-index: 1;
+
+    &::-ms-expand {
+      display: none;
+    }
+  }
+
+  &__icon {
+    fill: currentColor;
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    pointer-events: none;
+    z-index: 2;
+    width: $mu150;
+    height: $mu150;
+    right: $mu100;
+    background: inherit;
+    vertical-align: middle;
+
+    .is-focus & {
+      fill: $color-select-focus-icon;
+    }
+
+    .is-disabled & {
```

Comment:
we would not use the native attributes ? 

We would need to add `is-disabled` on the wrapper and `disbaled="true"` on the select, 
same goes for focus ? using javascript ? 

Seems quite complicated to implement safely and making sure that developper are not making mistakes. As well as making sure that accessibility is still OK for screen readers.
 
This all thing does not seem like a good solution at all. 

I think if we can't customize the native select, we should better be going for a native, non-customized one.